.github/PULL_REQUEST_TEMPLATE/pull_request_template.md

```diff
@@ -0,0 +1,18 @@
+# Issue
+
+https://github.com/SAP/open-ux-tools/issues/ _FILL IN ISSUE NUMBER HERE_
+
+## Description
+
+_FILL IN DETAILS HERE_
+
+## Checklist for Pull Requests
+
+- [ ] Supplied as many details as possible on this change.
+- [ ] Included the link to the associated issue in the [issues](https://github.com/SAP/open-ux-tools/issues)
+- [ ] The code is **easy to read and maintainable** by others.
+- [ ] The code conforms to the general development principles.
+- [ ] Functions and variables are suitably named and are sufficiently descriptive.
+- [ ] The code has **unit tests** where applicable and is easily unit-testable.
+- [ ] This PR is associated with a discrete task (smaller discrete tasks can be more efficiently reviewed).
```

Comment:
I think this needs more explanation or I'm not even sure if it belongs to this checklist. 
```suggestion
```

Reply:
Agreed. Removed it now https://github.com/SAP/open-ux-tools/pull/321/files#diff-45659dae3c8acfb7058b31c52c8600ca9c72a11a482372feea9ab85cb0f270fa